Json.More/JsonSerializerOptionsExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Json.More;
+
+// TODO: .NET 5+ would have these methods marked with `RequiresUnreferencedCodeAttribute` to warn against tree trimming
+/// <summary>
+/// Provides extension functionality for <see cref="JsonSerializerOptions"/>.
+/// </summary>
+public static class JsonSerializerOptionsExtensions
+{
+	/// <summary>
+	/// Returns the converter for the specified type.
+	/// </summary>
+	/// <typeparam name="T">The <see cref="Type"/> to convert.</typeparam>
+	/// <param name="options">The <see cref="JsonSerializerOptions"/> being used.</param>
+	/// <returns>An implementation of <see cref="JsonConverter<>"/> as determined by the provided options</returns>
+	public static JsonConverter<T> GetConverter<T>(this JsonSerializerOptions options)
```

Comment:
Is this used outside of this class?  I don't see it.  If not, it can be internal.

Reply:
Yeah, though it could be useful; there are plenty of reasons to use the converter.

For instance, the `HandleNull` property is supposed to be used to determine whether or not to write out `null`. This may be the expected option to avoid the `SchemaKeywordRegistry.GetNullValuedKeyword` situation - I'm not exactly sure what caused the need for that!

Reply:
That actually looks like it might legacy code that isn't used anymore.  I'd have to dig into older versions to figure out why I needed it.  I might still be needed and that it doesn't use it may be broken.